Automattic/woocommerce-connect-client
# Add Settings View for Payment Method Selection

Fixes #471 

To test:
First, add credentials to your local server, in local.json, e.g.:

```
        "dummy_auth_credentials": {
                "authenticated": true,
                "blog_id": 123456,
                "blog_owner_id": 123456
        },
```

you'll have to use real blog_id and blog_owner_ids, of course.

Then, open your local client and go to wp-admin > WooCommerce > Settings > WooCommerce Connect
Verify you see all your stored payment methods
Verify the save button enables if you change the selection
Verify the save button disables while saving

Caveats:
I'm still trying to get webpack + svgs to work (there are supposed to be card logos. Will fix before I merge this to master.

Note:
I created a new lib, form-base, that contains actions and reducers for forms like this. I did not want to deep link from the shared-settings container into the settings container for its state management.  We can use this new lib for other forms.

I also make the shared settings root view where the notices actions are connected, since that way the root view can control the text that appears (e.g. "Payment methods updated.") more appropriately.
